samples/vue-query/src/api/transformer/add-version.js

```diff
@@ -4,33 +4,41 @@
  * @param {OpenAPIObject} schema
  * @return {OpenAPIObject}
  */
-module.exports = (inputSchema) => ({
-  ...inputSchema,
-  paths: Object.entries(inputSchema.paths).reduce(
-    (acc, [path, pathItem]) => ({
-      ...acc,
-      [`v{version}${path}`]: Object.entries(pathItem).reduce(
-        (pathItemAcc, [verb, operation]) => ({
-          ...pathItemAcc,
-          [verb]: {
-            ...operation,
-            parameters: [
-              ...(operation.parameters || []),
-              {
-                name: 'version',
-                in: 'path',
-                required: true,
-                schema: {
-                  type: 'number',
-                  default: 1,
-                },
-              },
-            ],
+module.exports = (inputSchema) => {
+  const transformedPaths = {};
+
+  Object.entries(inputSchema.paths).forEach(([path, pathItem]) => {
+    const transformedPathItem = {};
+
+    if (path === '/api/v1/user/logout') {
```

Comment:
Added this condition

The rest is just refactoring from `reduce` to `forEach` because it was hard to read and modify, see more https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-array-reduce.md